### ISSUE TITLE
mat-icon-button touch target size is too small

### ISSUE THREAD

**Issue by mhebrard** [angular/components#14146]
I might be wrong but If I refers to the Material Design Guideline, a touchable button should be 48px large. When I use ```mat-icon-button``` the resulting element is only 40x40 ...
Is that not a problem ?

#### What are the steps to reproduce?
Providing a StackBlitz reproduction is the *best* way to share your issue. <br/>
StackBlitz starter: https://goo.gl/wwnhMV<br/>

#### What is the use-case or motivation for changing an existing behavior?

```
<button mat-icon-button>
  <mat-icon>favorite</mat-icon>
</button>
```

**Comment by amysorto** [comment]
The new MDC based icon button component that will be available in v15 soon is 48x48px. Closing this bug.
